### PR TITLE
Highlight pinned events and fix tooltip component

### PR DESCRIPTION
**Pull request by erlingfn**

### CODE REVIEW

**Comment by erlingfn** [comment]
Screenshot from 2019-03-11 22-32-51

Screenshot from 2019-03-11 22-37-18

File: app/routes/overview/components/CompactEvents.js [hunk 1]
```diff
@@ -36,6 +37,11 @@ export default class CompactEvents extends Component<Props> {
               <i className="fa fa-circle" />
             </span>
             <Link to={`/events/${event.id}`}>{event.title}</Link>
+            {event.pinned && (
+              <Tooltip content="Dette arrangementet er festet til forsiden">
+                <i className="fa fa-thumb-tack" />
```
Comment (SmithPeder) [inline app/routes/overview/components/CompactEvents.js]: How about some quick `CSS` magic ? 💯 
You can give it some `px` margin, and a quick transform, maybe slap on some color 🔥 

```js
style={{
  transform: 'rotate(-20deg)',
  marginRight: '4px',
  color: '#BE1600'
}}
```
 
image

Comment (erlingfn) [reply app/routes/overview/components/CompactEvents.js]: Yeah that works out great!

**Comment by odinuge** [comment]
Looking gooood :sunglasses: :

**Pull request merged**